lvgl/Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords = ["littlevgl", "lvgl", "graphical_interfaces"]
 build = "build.rs"
 
 [dependencies]
-lvgl-sys = { version = "0.6.2", path = "../lvgl-sys" }
+lvgl-sys = { version = "0.6.2", path = "../lvgl-sys", features = ["library"]}
```

Comment:
_Tiny_ nit; here (and line 80) add a space before the curly brace for stylistic consistency